
2021-10-29/20:27:22		All the required Files are initialized ....

2021-10-29/20:27:23		Entered into the class : CreateGoodTraining_table ....

2021-10-29/20:27:24		Good_trainingTable  created Successfully !....

2021-10-29/20:27:28		All the required Files are initialized ....

2021-10-29/20:27:28		Entered into the class : CreateGoodTraining_table ....

2021-10-29/20:27:29		Good_trainingTable  created Successfully !....

2021-10-29/22:03:05		All the required Files are initialized ....

2021-10-29/22:03:06		Entered into the class : CreateGoodTraining_table ....

2021-10-29/22:03:07		Good_trainingTable  created Successfully !....

2021-10-29/22:03:17		Excepation : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/00:19:22		All the required Files are initialized ....

2021-10-30/00:19:23		Entered into the class : CreateGoodTraining_table ....

2021-10-30/00:19:24		Good_trainingTable  created Successfully !....

2021-10-30/00:19:32		Excepation : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('54.196.58.228', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/00:58:17		Excepation : Unable to connect to the metadata service at https://3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29080/metadata. Check the cluster status in the cloud console. 

2021-10-30/00:58:26		Excepation : ('Unable to connect to any servers', ['3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com'])

2021-10-30/00:59:10		Excepation : Unable to connect to the metadata service at https://3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29080/metadata. Check the cluster status in the cloud console. 

2021-10-30/00:59:22		All the required Files are initialized ....

2021-10-30/00:59:23		Entered into the class : CreateGoodTraining_table ....

2021-10-30/00:59:24		Good_trainingTable  created Successfully !....

2021-10-30/01:54:11		All the required Files are initialized ....

2021-10-30/01:54:12		Entered into the class : CreateGoodTraining_table ....

2021-10-30/01:54:13		Good_trainingTable  created Successfully !....

2021-10-30/01:54:15		Exception : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OperationTimedOut('errors=None, last_host=None'), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/02:35:33		Exception : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('54.196.58.228', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/02:38:29		All the required Files are initialized ....

2021-10-30/02:38:31		Entered into the class : CreateGoodTraining_table ....

2021-10-30/02:38:32		Good_trainingTable  created Successfully !....

2021-10-30/02:38:32		Exception : ('Unable to connect to any servers', ['3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com'])

2021-10-30/02:39:51		Exception : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('54.196.58.228', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/02:40:00		Exception : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OperationTimedOut('errors=Timed out creating connection (5 seconds), last_host=None'), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('3.219.47.245', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/02:42:18		All the required Files are initialized ....

2021-10-30/02:42:19		Entered into the class : CreateGoodTraining_table ....

2021-10-30/02:42:19		Good_trainingTable  created Successfully !....

2021-10-30/20:43:52		All the required Files are initialized ....

2021-10-30/20:43:52		Entered into the class : CreateGoodTraining_table ....

2021-10-30/20:43:53		Good_trainingTable  created Successfully !....

2021-10-30/20:44:17		Exception : ('Unable to connect to any servers', ['3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com'])

2021-10-30/20:44:42		All the required Files are initialized ....

2021-10-30/20:44:42		Entered into the class : CreateGoodTraining_table ....

2021-10-30/20:44:43		Good_trainingTable  created Successfully !....

2021-10-30/20:45:09		Exception : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('3.219.47.245', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/20:45:39		Exception : ('Unable to connect to any servers', {'3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:acfdc860-4251-42a1-bceb-9a7667a1e06d': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:82243265-9b8b-423d-84c7-901973dfb47c': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out"), '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29042:6a4dd0c4-c94c-4861-b30a-ed2fb864d7f1': OSError(None, "Tried connecting to [('18.207.107.113', 29042)]. Last error: timed out")})

2021-10-30/20:46:59		Exception : Unable to connect to the metadata service at https://3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com:29080/metadata. Check the cluster status in the cloud console. 

2021-10-30/20:47:37		Exception : ('Unable to connect to any servers', ['3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com', '3db55841-4a14-4c14-9b42-7d57a84a4ae5-us-east-1.db.astra.datastax.com'])

2021-10-30/20:49:01		All the required Files are initialized ....

2021-10-30/20:49:02		Entered into the class : CreateGoodTraining_table ....

2021-10-30/20:49:03		Good_trainingTable  created Successfully !....

2021-10-30/20:49:05		All the required Files are initialized ....

2021-10-30/20:49:05		Entered into the class : CreateGoodTraining_table ....

2021-10-30/20:49:06		Good_trainingTable  created Successfully !....
